tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2126,6 +2178,136 @@ TEST_P(OpConverter_FP32_Test, ConvertTranspose) {
+                {0, 1, 2, 3, 4, 5,
+                 0, 1, 2, 3, 4, 5}},  // expected_results
+      TileParam{{1, 2, 3},            // input_dims
+                {1, 1, 2},            // multiplier
+                {0, 1, 2, 3, 4, 5},   // tensor
+                {1, 2, 6},            // expected_output_dims
+                {0, 1, 2, 0, 1, 2,
+                 3, 4, 5, 3, 4, 5}},  // expected_results
+      TileParam{{1, 2, 3},            // input_dims
+                {1, 2, 2},            // multiplier
+                {0, 1, 2, 3, 4, 5},   // tensor
+                {1, 4, 6},            // expected_output_dims
+                {0, 1, 2, 0, 1, 2, 3, 4, 5, 3, 4, 5,
+                 0, 1, 2, 0, 1, 2, 3, 4, 5, 3, 4, 5}} // expected_results
+  };
+
+  for (bool multiplier_is_tensor : {true, false}) {
+    for (bool input_is_tensor : {true, false}) {
+      for (auto p : test_params) {
+        std::vector<int>num_mults = {p.multiplier.size()};
+        std::vector<std::vector<int>>partial_input_dims_options = {{}};
+        if (multiplier_is_tensor) {
+          if (trt_mode_ == TrtTestMode::kImplicitBatch) {
+             p.status = Status(error::INVALID_ARGUMENT,
+                             "Conversion for Tile is not implemented for multipliers "
+                             "passed as a tensor in implicit batch mode");
+             num_mults = {1, p.multiplier.size()};
+          } else {
+            if (p.test_ID == 1) {
+              // Replacement of statuses, since when multiplier is a tensor AND
+              // trt_mode_ != TrtTestMode::kImplicitBatch we cannot define these
+              // statuses in ConvertTile::Validate() for first two tests from test_params
+              p.status = Status::OK();
+              p.runtime_status = Status(error::INTERNAL, "Incorrect number of profile config parameters");
+            }
+
+            if (trt_mode_ == TrtTestMode::kDynamicShape) {
+              partial_input_dims_options = {{}, num_mults};
+            }
+          }
+        }
+
+        for (auto partial_input_dims : partial_input_dims_options) {
+          if (multiplier_is_tensor && trt_mode_ != TrtTestMode::kImplicitBatch) {
+            if (trt_mode_ == TrtTestMode::kDynamicShape) {
+              if (p.test_ID != 1) {
+                p.runtime_status = partial_input_dims.empty()
+                  ? Status(error::INTERNAL, "Failed to build TensorRT engine")
+                  : Status::OK();
+                p.status = Status::OK();
+              }
+            }
+
+            if (p.test_ID == 2 && (trt_mode_ == TrtTestMode::kExplicitBatch || !partial_input_dims.empty())) {
+              p.status = Status(error::INVALID_ARGUMENT, "When replications are defined as a tensor, "
+                                                         "the number of elements in it must be equal "
+                                                         "to the rank of the input tensor. Got 4 vs 3.");
```

Comment:
This error message has a "different style" comparing with the rest, such as the one in line 2219. Can you make this one similar to the one in line 2219, by putting the values "4" and "3" right next to their description? "4" should go with "the number of elements (4)", for example.